### PR TITLE
React-Redux Capstone Project

### PR DESCRIPTION
**Pull request by Paul-js-hub**
### In this project I:

- Setup the project files.
-  Displayed a list of countries and filtered them.
- Created the tests for the application.
- Styled the application with React Bootstrap.
- Deployed the project to Heroku and tested for final details.
- Recorded a video for your project.
- Created a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Live link
+[Live Demo Link](https://africa-metadata.herokuapp.com/)
+
+## Live link
+[Video Link](https://www.loom.com/share/d1348d4f96ba4a498083422926882105)
```
Comment (rloterh) [inline README.md]: Good start with video presentation, however, the code portions of the presentation are missing. Kindly ensure that your video presentation includes the following:
- [ ] The video contains a demo of the project, highlighting all the project features.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Countries from './components/Countries/Countries';
+import Header from './components/Header/Header';
+import CountryData from './components/CountryData/CountryData';
+import './App.css';
+
+function App() {
```
Comment (rloterh) [inline src/App.js:8]: - [ ] Deployed app is displaying correctly, however the repository version compiles successfully but is displaying blank screen. Kindly check and update the version of your code on the repository accordingly, and the next code reviewer will pick it from there.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Paul, 

You did extremely great on this project! I really like your project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- React Router properly implemented
- State management properly implemented
- Category filter properly implemented


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+.color1 {
+  background-color: #dc4782 !important;
+}
+
+.country-details {
+  color: white;
+}
+
+.country-details:nth-child(odd) {
```
Comment (elisha2kyakpo1) [inline src/App.css:87]: - [ ] Kindly work on the styling, try as much as possible to match the project design especially on the home page. This is a mobile version of the app, let's pay attention to that as well.

File: README.md (hunk 1 above)
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Just like the previous reviewer said, the video presentation is too short and doesn't contain other presentation requirements. Kindly record another video and highlight features of the project (1) A piece of code you are proud of. (2) The video should contains a clear and objective description of the project in professional English. The topic and purpose of the project should be made clear. (3) The video should contains a demo of the project, highlighting all the project features.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Paul-js-hub, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- The app was deployed properly and it works ✔️
- Descriptive commit messages✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @Paul-js-hub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewers

## Optional suggestions
- Consider removing the redux logger from the production app. It consoles important data to the console which is not a good practice.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**